extension/demo/src/modules/constants.js

```diff
@@ -2,5 +2,8 @@
 
 const env = process.env.TARGET_ENV
 
-export const STORAGE_KEY = 'auth'
+export const ACCESS_TOKEN = 'accessToken'
+export const SUBSCRIPTION = 'subscription'
+export const PAYLOAD = 'payload'
+export const TTL = 'expiresAt'
```

Comment:
Maybe prefix them with something so it's clear what they're used for.

Though, frankly, it would probably be more useful to turn the whole storage module into a class/object with getters and setters for each (`getAccessToken`, `setAccessToken`, etc).

The latter might be better to do in a separate PR though.

Reply:
https://github.com/flattr/browser-extension-auth-flow-demo/issues/30